### ISSUE TITLE
Restarting after entering correct password can print confusing "error"

### ISSUE THREAD

**Issue by aaaidan** [HomeSpan/HomeSpan#6]
When the device restarts after entering the correct wifi password, there are usually a couple of messages about "Can't connect".  This state lasts long enough that it has tricked me into trying again with different (incorrect!) credentials. This seems to be more about the device itself not being ready rather than the password being incorrect. 

Is there a way to tell between "can't connect yet" and "wrong password"? Or perhaps rephrasing the message to "waiting to connect" or similar.

```
...
Accessory configuration number: 1

Connecting to: XXX... Can't connect. Re-trying in 5 seconds. Type 'X <return>' or press Control Button for 3 seconds to terminate search and delete WiFi credentials...
Connecting to: XXX... Can't connect. Re-trying in 5 seconds. Type 'X <return>' or press Control Button for 3 seconds to terminate search and delete WiFi credentials...
Success!  IP:  10.0.0.239

Starting MDNS...
Broadcasting as: HomeSpan-7BFF4D99B831.local (HomeSpan LightBulb / HomeSpan-ESP32)
...
```

**Comment by HomeSpan** [comment]
Thanks for the feedback on this.  Though there is a way to read the error message back from a WiFi hub to determine why the connection was rejected, it does not seem to work all the time, and is not universal.  However, since it usually takes 1-3 tries before connecting even with the right password, I'll update the messages (and timing of the messages) to let the user know to wait while HomeSpan is still in the "connecting" process.  I'll incorporate into the next release.

**Comment by HomeSpan** [comment]
Fixed in version 1.1.2
